tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/Serverless.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="Serverless.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation
+{
+    internal static class Serverless
+    {
+        private const string HandlerEnvName = "_HANDLER";
+        private const string FunctionEnvame = "AWS_LAMBDA_FUNCTION_NAME";
+        private const string IntegrationType = "Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS.Lambda";
+
+        internal static bool IsRunningInLambda()
+        {
+            return !string.IsNullOrEmpty(Environment.GetEnvironmentVariable(FunctionEnvame));
+        }
+
+        internal static NativeCallTargetDefinition BuildLambdaHandlerDefinition(string assemblyFullName)
+        {
+            LambdaHandler handler = new LambdaHandler(Environment.GetEnvironmentVariable(HandlerEnvName));
+            return new NativeCallTargetDefinition(handler.GetAssembly(), handler.GetFullType(), handler.GetMethodName(), handler.BuidParamTypeArray(), 1, 0, 0, 5, 65535, 65535, assemblyFullName, IntegrationType);
```

Comment:
question: I don't know how the lambda passes this info in HandlerEnvName, but doesn't the `assemblyFullName` contains the version of the assembly? If that's the case I think you can parse it with `AssemblyName` and use it to create a version range here.